src/OrchardCore.Modules/OrchardCore.Forms/Providers/IsByteLengthProvider.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text;
+using System.Threading.Tasks;
+using OrchardCore.Forms.Helpers;
+using OrchardCore.Forms.Services;
+using OrchardCore.Forms.Services.Models;
+
+namespace OrchardCore.Forms.Providers
+{
+    public class IsByteLengthProvider : IValidationRuleProvider
+    {
+        public int Index => 6;
+        public string DisplayName => "Is ByteLength";
+        public string Name => "IsByteLength";
+        public bool IsShowOption => true;
+        public string OptionPlaceHolder => "{&quot;min&quot; : 0, &quot;max&quot; : 20}";
```

Comment:
If it contains Html this should be an HtmlString, to handle encoding correctly, and set expectations.